teamcity/datadog_checks/teamcity/metrics.py

```diff
@@ -0,0 +1,224 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+from copy import deepcopy
+
+STANDARD_PROMETHEUS_METRICS = {
```

Comment:
Question: will I have to define something like this for rabbitmq as well?

Reply:
Not necessarily. The OM check just needs a `METRIC_MAP` dictionary of raw metric names mapped to the DD metric names. I created separate constants for each metric type for organization purposes, but it's not necessary.

Reply:
understood, thanks!